files/en-us/web/css/anchor/index.md

```diff
@@ -110,18 +108,18 @@ The CSS {{glossary("inset properties")}} that accept an `anchor()` function as a
 
```

Comment:
This section is a fantastic addition, but I found it a little overwhelming, so I tried to make it a bit more basic.